Mathlib/Algebra/Group/Defs.lean

```diff
@@ -305,6 +305,74 @@ theorem mul_right_cancel_iff : b * a = c * a ↔ b = c :=
 
 end IsRightCancelMul
 
+/-- A mixin for common left multiples. -/
```

Comment:
I think this docstring could be more informative. I have suggested one way to change it, but you should feel free to do something different.
```suggestion
/-- We say that `G` has *left common multiples* if for all `a b : G`, there exist
`c d : G` such that `c * a = d * b`. This mixin carries both the property
of having left common multiples and the data of how to determine `c` and `d` from `a` and `b`.
This will be used to define the left Ore localization of a monoid with respect to itself. -/
```

Reply:
I like it, thanks!